plugins/externalwrench/src/ApplyExternalWrench.cc

```diff
@@ -142,16 +155,18 @@ void RPCServerThread::run()
         m_rpcPort.read ( command,true );
         if ( command.get ( 0 ).asString() == "help" ) {
             this->m_reply.addVocab ( yarp::os::Vocab::encode ( "many" ) );
-            this->m_reply.addString ( "The defaul operation mode is with single wrench" );
+            this->m_reply.addString ( "The default operation mode is with single wrench" );
             this->m_reply.addString ( "Insert [single] or [multiple] to change the operation mode" );
+            this->m_reply.addString ( "The default frame orientation is the one of the base/root frame" );
```

Comment:
It  is not clear that "base/root frame" correspond to `orient_global`, so I would modify as: 
```suggestion
            this->m_reply.addString ( "The default frame orientation is the one of the base/root frame ([orient_global])" );
```